pages/restake/lst-concepts.mdx

```diff
@@ -1,41 +1,53 @@
 
-LSTs can be used as collateral in Tangle's restaking protocols. Users can re-lock their LSTs in new service offerings, enhancing security and slashing mechanisms.
+### 3. **Unbonding**
 
-### Reward Distribution
+- Users can **unbond** their staked tokens with the `unbond` function, which allows them to withdraw up to a specified amount of their staked assets. The system ensures that rewards are automatically collected during the unbonding process to prevent forfeiture.
+- **Permissionless unbonding**: Under specific conditions, such as when a pool is blocked or destroying, users can unbond their tokens permissionlessly.
 
-Tangle's reward system deducts a fee from the underlying staking rewards to support the protocol. The remaining rewards are distributed proportionally to liquid stakers.
+### 4. **Withdrawals**
 
-### Liquidity Challenges
+- **Withdraw Unbonded**: Once tokens are unbonded, users can withdraw their funds using the `withdraw_unbonded` function. This function can also be used permissionlessly if certain conditions are met (e.g., the pool is destroying, or the caller is the depositor and the only member left).
 
-Operation-specific LSTs may face liquidity issues in the DeFi ecosystem. Tangle addresses this by encouraging integrations that support trading between LSTs from the same underlying protocol.
+### 5. **Pool Management**
 
-### Security and Slashing
+- **Create Pools**: Users can create new delegation pools with the `create` function. A pool requires an initial deposit, and the creator can set the roles for root, nominator, and bouncer. Pools can also be created with a previously used pool ID using the `create_with_pool_id` function.
+- **Pool State Management**: The state of a pool can be modified using the `set_state` function. Pools can have various states, such as open, blocked, or destroying. Once a pool enters the destroying state, it cannot change its state again.
 
-LSTs serve as collateral for outsourced security and slashing. This ensures network security while enabling innovative staking and restaking solutions.
+### 6. **Rewards and Commission**
 
-### Multi-Chain Support
+- **Commission**: Pools can set a commission percentage using the `set_commission` function. This determines the amount of rewards that are paid to the pool's commission account.
+- **Claiming Commission**: Pending commission can be claimed using the `claim_commission` function, which adds the claimed amount to the total claimed commission and resets pending commission to zero.
+- **Maximum Commission**: The maximum allowable commission can be set with the `set_commission_max` function, ensuring that future commissions are bound by this limit.
+- **Commission Change Rate**: The rate at which commission changes can be controlled using the `set_commission_change_rate` function.
```

Comment:
Not sure I follow why this exists? Can we provide examples too? Does this mean there's a value that defines how much the commission can change from 1 value to another? How is it represented and why is that needed?

Reply:
yes the goal is to prevent a pool creator to advertise 1% commission and after getting deposits to set 10%, this is from the nomination pools pallet, once the upper bound is set, it can only go lower not higher, from docs

```
/// Once a change rate configuration has been set, only more restrictive values can be set
	/// thereafter. These restrictions translate to increased `min_delay` values and decreased
	/// `max_increase` values.``